plugin-gradle/src/main/java/com/diffplug/gradle/spotless/FormatExtension.java

```diff
@@ -743,12 +743,12 @@ protected Project getProject() {
 	 * Creates an independent {@link SpotlessApply} for (very) unusual circumstances.
 	 *
 	 * Most users will not want this method.  In the rare case that you want to create
-	 * a `SpotlessApply` which is independent of the normal Spotless machinery, this will
+	 * a {@code SpotlessApply} which is independent of the normal Spotless machinery, this will
 	 * let you do that.
 	 *
-	 * The returned task will not be hooked up to the global `spotlessApply`, and there will be no corresponding `check` task.
+	 * The returned task will not be hooked up to the global {@code spotlessApply}, and there will be no corresponding {@code check} task.
 	 *
-	 * NOTE: does not respect the rarely-used [`spotlessFiles` property](https://github.com/diffplug/spotless/blob/b7f8c551a97dcb92cc4b0ee665448da5013b30a3/plugin-gradle/README.md#can-i-apply-spotless-to-specific-files).
+	 * NOTE: does not respect the rarely-used [{@code spotlessFiles} property](https://github.com/diffplug/spotless/blob/b7f8c551a97dcb92cc4b0ee665448da5013b30a3/plugin-gradle/README.md#can-i-apply-spotless-to-specific-files).
```

Comment:
I wonder if this link should also be changed to use `<a href="...">...</a>` syntax. Thoughts?

Reply:
Good call, same a bove.